src/feature-flags/flags.ts

```diff
@@ -105,6 +105,17 @@ const InternalFLAGS = {
       visibleInFeatureFlagPanel: true,
     },
   },
+  'components-page': {
+    key: 'components-page',
+    description: 'Show Components page',
```

Comment:
I'm wondering if it wouldn't be better to have a more "explanatory" description? "Show Components page" might be misleading to the user, since they can think it relates to the Components tab that already exists.

How about name it to something like: `"New Application / Component model - Components page"`? :)